### ISSUE TITLE
Closure compilation failed

### ISSUE THREAD

**Issue by JarrodCTaylor** [lambdaisland/deep-diff2#17]
When attempting to do a minified build the following error is thrown

```
Closure compilation failed with 1 errors
--- lambdaisland/deep_diff2/puget/color/html.cljc:42
Cannot convert ECMASCRIPT_2018 feature "RegExp Lookbehind" to targeted output language.
```

This seems to be the culprit: 
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/4416952/78509112-4e6c5400-7751-11ea-9241-f6117ca6f3e1.png">

**Comment by plexus** [comment]
This is code we inherited from Puget, but inlined when porting it to CLJC. Lookbehind is indeed a regex feature that is not universally available in JS engines yet.

A fix should be fairly easy though. This is really just trying to split a string that starts with an entity (e.g. `&lt;` or `&nbsp;`) into the entity and the rest of the string. This could be done with `subs` and `clojure.string/index-of`.

Here's a sketch of what that could look like (untested)

``` clojure
(let [semicolon-pos (str/index-of span \;)
      escaped (subs span 0 semicolon-pos)
      span (subs span semicolon-pos)]
  ,,,)
```

Would you be interested in putting together a PR for this?

**Comment by JarrodCTaylor** [comment]
I will take a pass at it this weekend.

**Comment by plexus** [comment]
![thankyou](https://user-images.githubusercontent.com/32212/78899068-27759100-7a75-11ea-8cdb-dbc25af2a389.gif)

**Comment by oxalorg** [comment]
Fixed in #18 Closing the issue. 

Please feel free to comment here if this is still an issue for someone else 😄
